src/scan_nm2.cc

```diff
@@ -329,12 +329,18 @@ void scan_nm2_actor(zsock_t* pipe, void* args)
 
     if (!args) {
         logError("{} : actor created without parameters", __FUNCTION__);
+        zmsg_t* reply = zmsg_new();
+        zmsg_pushstr(reply, REQ_DONE);
+        zmsg_send(&reply, pipe);
```

Comment:
But here is 
```cpp
   AutoRemove cleanup([&]() {
        if (args) {
            zlist_t* argv = static_cast<zlist_t*>(args);
            zlist_destroy(&argv);
        }
        zmsg_t* reply = zmsg_new();
        zmsg_pushstr(reply, REQ_DONE);
        zmsg_send(&reply, pipe);
    });
```
    Which is executed on leaving the function scope, e.g, in return.

Reply:
Seems to be another issue with blocking thread pool.